### DISCUSSION TITLE
Freetext option in dropdown

### DISCUSSION THREAD

**Discussion by psinger** [h2oai/wave#1018] [Q&A]
Is it possible to have an additional freetext option in a dropdown without a separate ui element?

So have a dropdown with a few options, and automatically a freetext option for the user to input.

**Comment by lo5** [comment]
There's a combobox component that allows typing in a value (in addition to preset choices): 
- https://wave.h2o.ai/docs/api/ui#combobox
- https://wave.h2o.ai/docs/examples/combobox

**Reply by psinger** [reply]
Thank you @lo5 - follow up question:

Would it be possible to also support multi-choice here? So I am thinking about the scenario where the user can select from preset choices, and can add additional own choices.

**Reply by lo5** [reply]
We can add multiple choice selections to the combobox.
https://github.com/h2oai/wave/issues/1034
